test/functional/test-xhr.js

```diff
@@ -663,6 +663,29 @@ describe('XHR', function() {
       });
     });
 
+    it('should be cloneable and each instance should provide text', () => {
+      const response = new FetchResponse(mockXhr);
+      const clone = response.clone();
+      return response.text()
+          .then(result => {
+            const callCloneText = () => clone.text().then(clonedText => {
+              expect(clonedText).to.equal(TEST_TEXT);
+            });
+            expect(callCloneText, 'should not throw').to.not.throw(Error);
```

Comment:
This wrapping isn't necessary. If the clone's `#text` were to throw syncly, it would cause the tests to fail (because the promise block it's inside is waited on by the test).